beacon-chain/sync/initial-sync/service.go

```diff
@@ -426,33 +427,36 @@ func (s *Service) fetchOriginColumns(roBlock blocks.ROBlock, maxAttempts int, de
 	root := roBlock.Root()
 
 	params := sync.DataColumnSidecarsParams{
-		Ctx:         s.ctx,
-		Tor:         s.clock,
-		P2P:         s.cfg.P2P,
-		CtxMap:      s.ctxMap,
-		Storage:     s.cfg.DataColumnStorage,
-		NewVerifier: s.newDataColumnsVerifier,
+		Ctx:                     s.ctx,
+		Tor:                     s.clock,
+		P2P:                     s.cfg.P2P,
+		CtxMap:                  s.ctxMap,
+		Storage:                 s.cfg.DataColumnStorage,
+		NewVerifier:             s.newDataColumnsVerifier,
+		DownscorePeerOnRPCFault: true,
```

Comment:
how does this get reset? feels strange to start with true?

Reply:
This `DownscorePeerOnRPCFault: true` is set in `params`, and `params` only lives in the `fetchOriginColumns` function. We specially want peers to be downscored on RPC fault in this function.

For all other usages of `sync.FetchDataColumnSidecars`, peers won't be downscored if `DownscorePeerOnRPCFault: true` is not specified. (And it turns out `DownscorePeerOnRPCFault: true` is only specified here.)